ecl/data_source_ecl_network_fic_gateway_v2_mock_test.go

```diff
@@ -0,0 +1,294 @@
+	mc.Register(t, "fic_gateway", "/v2.0/fic_gateways", testMockNetworkV2FICGatewayListStatusQuery)
+	mc.Register(t, "fic_gateway", "/v2.0/fic_gateways", testMockNetworkV2FICGatewayListTenantIDQuery)
+
+	mc.StartServer(t)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheckFICGateway(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccNetworkV2FICGatewayDataSourceDescription,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNetworkV2FICGatewayDataSourceID("data.ecl_network_fic_gateway_v2.fic_gateway_1"),
+				),
+			},
+			resource.TestStep{
+				Config: testAccNetworkV2FICGatewayDataSourceFICServiceID,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNetworkV2FICGatewayDataSourceID("data.ecl_network_fic_gateway_v2.fic_gateway_1"),
+				),
+			},
+			resource.TestStep{
+				Config: testAccNetworkV2FICGatewayDataSourceID,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNetworkV2FICGatewayDataSourceID("data.ecl_network_fic_gateway_v2.fic_gateway_2"),
+				),
+			},
+			resource.TestStep{
+				Config: testAccNetworkV2FICGatewayDataSourceID2,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNetworkV2FICGatewayDataSourceID("data.ecl_network_fic_gateway_v2.fic_gateway_2"),
+				),
+			},
+			resource.TestStep{
+				Config: testAccNetworkV2FICGatewayDataSourceName,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNetworkV2FICGatewayDataSourceID("data.ecl_network_fic_gateway_v2.fic_gateway_1"),
+				),
+			},
+			resource.TestStep{
+				Config: testAccNetworkV2FICGatewayDataSourceQoSOptionID,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNetworkV2FICGatewayDataSourceID("data.ecl_network_fic_gateway_v2.fic_gateway_1"),
+				),
+			},
+			resource.TestStep{
+				Config: testAccNetworkV2FICGatewayDataSourceStatus,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNetworkV2FICGatewayDataSourceID("data.ecl_network_fic_gateway_v2.fic_gateway_1"),
+				),
+			},
+			resource.TestStep{
+				Config: testAccNetworkV2FICGatewayDataSourceTenantID,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNetworkV2FICGatewayDataSourceID("data.ecl_network_fic_gateway_v2.fic_gateway_1"),
+				),
+			},
+		},
+	})
+}
+
+var testMockNetworkV2FICGatewayListDescriptionQuery = fmt.Sprintf(`
+request:
+    method: GET
+    query:
+      description:
+        - " "
+response:
+    code: 200
+    body: >
+        {
+            "fic_gateways": [
+                {
+                    "description": "",
+                    "fic_service_id": "66a63898-32a5-4b9d-8925-f52be1d84764",
+                    "id": "fc546cf7-1956-436b-a9b4-edc917e397cf",
+                    "name": "F032000001492",
+                    "qos_option_id": "d384d7f5-22aa-46e5-8cf5-759e87c7b2fd",
+                    "status": "ACTIVE",
+                    "tenant_id": "%s"
```

Comment:
以下のように書いたほうが簡潔だと思います。
```suggestion
                    "tenant_id": %q
```